### PR TITLE
Make it possible to simulate the game server being online

### PR DESCRIPTION
**Pull request by InoUno**
* Added a small server that can be run locally, which just uses the game server port, in order to make the port scanner test for "Eden is online" pass.
* Changed the game server port/host to be configurable via the `.env` file.

### CODE REVIEW

**Comment by InoUno** [comment]
I like it! At least for now it will probably do the trick quite nicely. 
Will add that in and omit the mock game server.

A few potential limitation of the solution, that I can think of right now: 
* If the game server is ever going to be queried for data, it would better to have an actual mock game server, that can reply correspondingly (i.e. healthiness of zones, local vars, etc).
* Probably not a huge issue, or an issue at all, but it won't be possible to take mock-Eden online/offline while developing without also rebooting the website server, since it's only set during startup.

File: api/v1/misc.js [hunk 1]
```diff
@@ -7,17 +7,23 @@ const scanner = require('portscanner');
...
-        scanner.checkPortStatus(54231, 'localhost', async (error, status) => {
-            if (!error && status === 'open') {
-                const query = 'SELECT COUNT(*) AS ct FROM accounts_sessions JOIN chars ON accounts_sessions.charid = chars.charid WHERE gmlevel = 0;';
-                const online = await req.app.locals.query(query);
-                return res.status(200).send(online[0].ct.toString());
-            }
-            res.status(404).send();
-        });
+        let gameServerOnline = false;
+        if (process.env.MOCK_GAME_SERVER_ONLINE == "true") {
```
Comment (slassen) [inline api/v1/misc.js:13]: Can we use strict equality here?

File: api/v1/misc.js [hunk 2]
```diff
@@ -7,17 +7,23 @@ const scanner = require('portscanner');
...
+            gameServerOnline = true;
+        } else {
+            const status = await scanner.checkPortStatus(process.env.GAME_SERVER_PORT, process.env.GAME_SERVER_HOST)
+            gameServerOnline = status == 'open';
```
Comment (slassen) [inline api/v1/misc.js:17]: Same here for equality.

File: example.env [hunk 3]
```diff
@@ -3,6 +3,9 @@ MYSQLUSER=MYSQLUSER
 MYSQLPASS=MYSQLPASS
 MYSQLDB=MYSQLDB
 MYSQLPORT=MYSQLPORT
+GAME_SERVER_HOST=localhost
+GAME_SERVER_PORT=54321
+MOCK_GAME_ONLINE=true
```
Comment (slassen) [inline example.env:8]: `MOCK_GAME_SERVER_ONLINE` is what's used in `api/v1/misc.js` but `MOCK_GAME_ONLINE` in the `example.env`.

File: api/v1/misc.js (hunk 1 above)
Comment (slassen) [reply api/v1/misc.js:13]: I know there was someone adding linting to the client, but for now no linting here. Can we also use single quotes here?

File: example.env (hunk 3 above)
Comment (InoUno) [reply example.env:8]: Woops, seems like I hadn't pressed save in my editor, so it wasn't included in the commit. Good catch.

**Pull request merged**